ui/ui-expenses/src/screens/prices/prices.jsx

```diff
@@ -35,38 +37,44 @@ const Prices = ({
 }) => (
   <Fragment>
     <div className={styles.block}>
-      <h2>Prix</h2>
-      <div className={styles.buttons}>
-        {prices.map(price => (
-          <Price key={price} price={price} selected={price === prices[0]} />
-        ))}
-      </div>
+      <Capture />
     </div>
+    {prices.length > 0 &&
```

Comment:
Please no.
You can do a route or something.

Reply:
or use the loader
or other solution than that

Reply:
in my mind is temporary, the `Price` screen will be refactor after.
But ok, I delete this condition.